Snowflake.Data.Tests/parameters.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
please don't commit this file

Reply:
I think we can commit, it's done the same way for i.e. jdbc and nodejs - it has standard params for local environment

Reply:
maybe it's something you can also ask on meeting about CASEC-9266 and make sure it's ok to have those files ?

Reply:
Let's resolve the question in `.gitignore` and it will solve the discussion here also